### PR TITLE
Expose sailthru key from ecomworker to ecommerce service

### PR DESCRIPTION
**Pull request by coryleeio**

### CODE REVIEW

**Comment by coryleeio** [comment]
This makes it so that ecomworker and ecommerce both read from the same sailthru key, and exposes that key to ecommerce.

File: playbooks/roles/ecomworker/defaults/main.yml [hunk 1]
```diff
@@ -113,8 +115,8 @@ ECOMMERCE_WORKER_SERVICE_CONFIG:
     SAILTHRU_ENROLL_TEMPLATE: '{{ ECOMMERCE_WORKER_SAILTHRU_ENROLL_TEMPLATE }}'
     SAILTHRU_ABANDONED_CART_TEMPLATE: '{{ ECOMMERCE_WORKER_SAILTHRU_ABANDONED_CART_TEMPLATE }}'
     SAILTHRU_ABANDONED_CART_DELAY: '{{ ECOMMERCE_WORKER_SAILTHRU_ABANDONED_CART_DELAY }}'
-    SAILTHRU_KEY: '{{ ECOMMERCE_WORKER_SAILTHRU_KEY }}'
-    SAILTHRU_SECRET: '{{ ECOMMERCE_WORKER_SAILTHRU_SECRET }}'
+    SAILTHRU_KEY: '{{ ECOMMERCE_SAILTHRU_KEY }}'
```
Comment (feanil) [inline playbooks/roles/ecomworker/defaults/main.yml]: Why did you make this change? These are two different services and should have different names for these settings.  Even if they do eventually take the same key as input.

**Review by feanil — CHANGES_REQUESTED** [review]
I don't think you should change anything about the ecommerce worker.

**Review by feanil — APPROVED** [review]

**Comment by coryleeio** [comment]
We talked and I agreed that we should not combine this setting.

**Pull request merged**